package.json

```diff
@@ -23,6 +23,7 @@
     "dotenv-webpack": "^8.0.1",
     "eslint": "^8.35.0",
     "eslint-config-prettier": "^8.6.0",
+    "node-sass": "^8.0.0",
```

Comment:
I think this package is deprecated
ref.) https://www.npmjs.com/package/node-sass